src/provider/controller/NotificationCenter.ts

```diff
@@ -69,7 +69,7 @@ export class NotificationCenter extends AsyncInit {
             throw error;
         }
         await this.hideWindowOffscreen();
-        this._trayIcon.setIcon('https://openfin.co/favicon-32x32.png');
+        this._trayIcon.setIcon(require('../../../res/provider/ui/image/shapes/trayIcon.png'));
```

Comment:
The require is required to be inline here, otherwise there is a great deal of work needed for unit tests to handle the image require outside of inline. I'm assuming the unit test is mocking it and handling it safely.